src/components/creator/Selection.tsx

```diff
@@ -40,23 +40,22 @@ const characters: CharacterCardProps[] = [
 	},
 ]
 
-const defaultScene = (type: SceneType): Scene => {
-	return {
-		type: type,
-		maps: [[['A', '-', '-'],
-		['-', '-', '-'],
-		['-', '-', '-']]]
-	}
-}
+export const defaultMaps: SceneMap[] = [[['A', '-', '-', '-'],['-', '-', '-','-'],['-', '-', '-','-'],['-','-','-','-']]]
 
 export const defaultChallenge = (type: SceneType): SerializedChallenge => {
+    if(!process.env.REACT_APP_CREATOR_VERSION) throw new Error("Missing Creator version. ENV not set")
+	const creatorVersion = parseInt(process.env.REACT_APP_CREATOR_VERSION!)
```

Comment:
@dlopezalvas 

Esto puede ser simplemente:
```ts
const creatorVersion = 1
```

Cuando cambiemos el formato de archivo, podemos pasar a la próxima versión y ya, cambiando el 1 por el 2. Si querés podés ponerlo en el mismo lugar donde está el SerializedChallenge, así ese número está definido al lado de donde está definida justamente la estructura que queremos versionar, junto con un `// remember to change creatorVersion on breaking changes`.